packages/fast-usdc/package.json

```diff
@@ -1,34 +1,64 @@
 {
-  "name": "fast-usdc",
-  "private": true,
-  "version": "0.1.1-u18.2",
-  "description": "Create an Agoric Javascript smart contract application",
+  "name": "@agoric/fast-usdc",
+  "version": "0.1.0",
+  "description": "CLI and library for Fast USDC product",
```

Comment:
@mhofman I've intentionally reset this to 0.1.0 because its now a public package (didn't really get published before)

Reply:
Also, the CHANGELOG file is not in the diff but should we also wipe it clean?

Reply:
I would avoid mucking with the versions here, and leave the previously auto-incremented one even if it wasn't published.